test/detect/index.js

```diff
@@ -34,7 +34,7 @@ function testAutoDetection(language, {detectPath}) {
 
 describe('hljs.highlightAuto()', () => {
   before( async function() {
-    let thirdPartyLanguages = await getThirdPartyLanguages();
+    let thirdPartyPackages = await getThirdPartyPackages(); // !! not used
```

Comment:
It's used on line 49.  `listLanguages` in the build has the FULL language list.  We only need to figure out where the tests are at.

Reply:
oops, was being blind

Reply:
Can you remove the not used comments?

Reply:
done.
Oh cool, the code snippit gets an \[Outdated] annotation above it!